x-pack/plugins/security_solution/public/management/pages/event_filters/store/middleware.ts

```diff
@@ -62,6 +82,94 @@ const eventFiltersCreate = async (
   }
 };
 
+const eventFiltersUpdate = async (
+  store: ImmutableMiddlewareAPI<EventFiltersListPageState, AppAction>,
+  eventFiltersService: EventFiltersService
+) => {
+  const submissionResourceState = store.getState().form.submissionResourceState;
+  try {
+    const formEntry = store.getState().form.entry;
+    if (!formEntry) return;
+    store.dispatch({
+      type: 'eventFiltersFormStateChanged',
+      payload: {
+        type: 'LoadingResourceState',
+        previousState: { type: 'UninitialisedResourceState' },
+      },
+    });
+
+    const sanitizedEntry: UpdateExceptionListItemSchema = transformOutput(
+      formEntry as UpdateExceptionListItemSchema
```

Comment:
This is because in some places we have this type: `UpdateExceptionListItemSchema | CreateExceptionListItemSchema` but in some others we specify the type to be `UpdateExceptionListItemSchema` or `CreateExceptionListItemSchema`
For example, state.form.entry is a `UpdateExceptionListItemSchema | CreateExceptionListItemSchema` type but the transformOutput function only allows a `UpdateExceptionListItemSchema | ExceptionListItemSchema` and returns the same type.
Then the `addNewComments` returns a `UpdateExceptionListItemSchema | CreateExceptionListItemSchema` but we need a `UpdateExceptionListItemSchema` for the `updateOne` service method.